platforms/documentation/docs/src/docs/userguide/reference/glossary.adoc

```diff
@@ -160,21 +212,28 @@ When invoked, the Gradle build executes a set of tasks based on the defined buil
 
 == M
 
+[[sub:terminology_maven_central]]
+MavenCentral::
+https://search.maven.org/[MavenCentral] is the main repository that hosts Maven publications.
+It is operated by a company named https://www.sonatype.com/[Sonatype] and is the default repository for a lot of the ecosystem.
++
+Many other repositories exists like (the now defunct) https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/[jcenter] or the https://maven.google.com/web/index.html[Google Maven repository].
+
 [[sub:terminology_module]]
 Module::
 A piece of software that evolves over time e.g., link:https://github.com/google/guava[Google Guava].
-Every module has a name. Each module release is optimally represented by a <<#sub:terminology_module_version, module version>>.
```

Comment:
🤔 did this `#` pass the dead link check? Consider investigating/fixing the check

Reply:
Yes, it works with or without the #.